src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -727,8 +818,8 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
             <Header sub>{t('create_single_item_modal.representation_label')}</Header>
             <Row>
               {this.renderRepresentation(BodyShapeType.BOTH)}
-              {this.renderRepresentation(BodyShapeType.MALE)}
-              {this.renderRepresentation(BodyShapeType.FEMALE)}
+              {!isSmartItem && this.renderRepresentation(BodyShapeType.MALE)}
```

Comment:
wdyt about adding this validation in the condition above? 

`isAddingRepresentation || isSmartItem ? null : ( ... )`

Reply:
This will be in the next PR with the real UI 😄